Puppetfile

```diff
@@ -9,6 +9,22 @@ mod 'example42/hieradata',
   :branch => :control_branch,
   :default_branch => 'production'
 
+# Puppet 6 Core Modues
```

Comment:
What happens if one uses this Puppetfile with Puppet pre 6?

Reply:
Good question. I wrote to puppet-users for feedback.

Reply:
https://groups.google.com/forum/#!msg/puppet-users/HMgdFb-cVy8/_JjyHSy7CAAJ
Puppet ensures compatibility.